frontend/src/app/components/landing-page/landing-page.component.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Component
+} from '@angular/core';
+import {
+  FormControl, FormGroup, Validators
+} from '@angular/forms';
+
+@Component({
+  selector: 'app-landing-page',
+  templateUrl: './landing-page.component.html',
+  styleUrls: ['./landing-page.component.sass']
+})
+export class LandingPageComponent {
+  showPassword: boolean = false;
+  isLogInClicked: boolean = false;
+  loginForm = new FormGroup({
+    login: new FormControl('', [Validators.required]),
+    password: new FormControl('', [
+      Validators.required,
+      Validators.minLength(6),
+      Validators.pattern('^[a-zA-Z0-9@._-]*$')
+    ])
+  });
+
+  constructor() {
+    this.loginForm.valueChanges.subscribe((changes) => {
+      this.isLogInClicked = false;
+    });
+  }
+
+  logInpOnClick() {
+    this.isLogInClicked = true;
+  }
+
+  toggleShowPassword() : void {
```

Comment:
There's no need to explicitly write return type. TS compiler understands the return type in 99% of cases